scripts/release/test-strip-appimage-rpaths.sh

```
@@ -0,0 +1,118 @@
+
+fail() { echo "[test-rpaths] FAIL: $1" >&2; exit 1; }
+
+# --- Case 1: sanitize_elf_rpaths strips a CI build-machine RPATH ------------
+APPDIR="$WORK/squashfs-root"
+mkdir -p "$APPDIR/usr/lib" "$APPDIR/shared/lib"
+
+# An ELF that mimics libcef.so with an absolute build-runner RUNPATH plus a
+# legitimate $ORIGIN entry — only the absolute one should be dropped.
+FAKE_CEF="$APPDIR/usr/lib/libcef.so"
+cp "$HOST_ELF" "$FAKE_CEF"
+patchelf --set-rpath '/home/runner/.cache/tauri-cef/x/shared/lib:$ORIGIN/../shared/lib' "$FAKE_CEF"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)